packages/hardhat-helpers/src/etherscan.js

```diff
@@ -4,6 +4,15 @@ const etherscan = {
   apiKey: {
     // xdai requires only placeholder api key
     xdai: 'api-key',
+    polygon: 'W9TVEYKW2CDTQ94T3A2V93IX6U3IHQN5Y3',
```

Comment:
I added these because I got tired of having to set them all the time. I don't think it's a problem to disclose them. We only use the API to verify contracts anyway.

Reply:
I use a .env file with all the api keys but agree i always forget to export it which leads to annoying failing steps in scripts